fastlane/jetpack_metadata/it/release_notes.txt

```diff
@@ -0,0 +1,3 @@
+Abbiamo disattivato i promemoria relativi al blog per i siti ospitati personalmente che non sono connessi a Jetpack, poiché questa funzionalità è disponibile solo nell'app Jetpack.
```

Comment:
Just calling out that this note could be confusing. The original English copy is:

https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/fastlane/jetpack_metadata/default/release_notes.txt#L1

The dependent clause might make sense in the WordPress app but I don't think it adds much in the Jetpack release notes. In fact, I'd argue it makes the entry confusing.

Still, I decided to submit the notes as they are to prioritize giving ourselves as much time as possible for review and shipping the changes. 

cc @twstokes as the author of #20970 which implemented the changes referred to here.

Reply:
👋 Thanks for surfacing this @mokagio!

If I understand correctly, editorial [took my release note here](https://github.com/wordpress-mobile/WordPress-iOS/pull/20970/files#diff-558ce8820a544da19f7fccdb2d1d661e46d68c130a1a39abdd222aa3484a5873R3) and tweaked it a bit, which I agree that it could add some confusion.

Perhaps I could've caught this by monitoring the appropriate channel, although I haven't done this in the past. (p1689251268473219-slack-C02S5P5MBGA)

Reply:
> If I understand correctly editorial [took my release note here](https://github.com/wordpress-mobile/WordPress-iOS/pull/20970/files#diff-558ce8820a544da19f7fccdb2d1d661e46d68c130a1a39abdd222aa3484a5873R3) and tweaked it a bit, which I agree that it could add some confusion.

I haven't checked with them, but that's what is likely to have happened.

> Perhaps I could've caught this by monitoring the appropriate channel, although I haven't done this in the past. (p1689251268473219-slack-C02S5P5MBGA)

You are welcome to do so, but I wouldn't expect anyone to. The process mostly takes care of itself and has been quite reliable and error-free so far.

I pinged you merely as an interested party. No followup expected 😄 

Incidentally, I think it's safe to assume that Blogging Reminders are only available on Jetpack, right? At least, that's what showed on my end today when I verified that on device for the 22.9 raw release notes.

Reply:
Sounds good, thanks @mokagio!

> Incidentally, I think it's safe to assume that Blogging Reminders are only available on Jetpack, right? At least, that's what showed on my end today when I verified that on device for the 22.9 raw release notes.

Correct! The [feature flag will only return `true`](https://github.com/wordpress-mobile/WordPress-iOS/blob/475340fbb94645397522009132cca1ca5b11a6c8/WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift#L54) if it's the Jetpack app.